_gsocorgs/2020/slac.md

```diff
@@ -0,0 +1,8 @@
+---
```

Comment:
SLAC doesn't have a logo?

Reply:
Yes, SLAC has lots of logos. Please find information on how to use them and logo links:
https://www6.slac.stanford.edu/about/logo-resources

Reply:
:+1: I have taken an existing logo inside the repository and renamed it SLAC-logo.png

Reply:
> 👍 I have taken an existing logo inside the repository and renamed it SLAC-logo.png

In which repository? We have to keep the old-named logos in order to show the previous years' GSoC pages …

Reply:
The same as above, please restore the previous logos which are in use.

Reply:
:+1: Done